### PR TITLE
Update Fields - Calculated Fields - Sample of user document 

### PR DESCRIPTION
**Pull request by recca5p**

### CODE REVIEW

File: docs/user/fields/calculated/samples.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 # Samples, Tips, and Useful Cases
 
 This topic includes the following information:
 
-- [Samples, Tips, and Useful Cases](#samples-tips-and-useful-cases)
+- [Samples, Tips, and Useful Cases](#)
   - [Basic Sample Expressions](#basic-sample-expressions)
   - [Converting Unix TimeStamps to Usable Dates](#converting-unix-timestamps-to-usable-dates)
   - [YoY Analysis: Comparing Revenue Figures for a 2-Year Period](#yoy-analysis-comparing-revenue-figures-for-a-2-year-period)
```
Comment (hainv-ohio) [inline docs/user/fields/calculated/samples.md:13]: Remove

**Review by hainv-ohio — CHANGES_REQUESTED** [review]
The file links in tables, change them to use local files instead

File: docs/user/fields/calculated/samples.md (hunk 1 above)
Comment (recca5p) [reply docs/user/fields/calculated/samples.md]: Removed

**Comment by recca5p** [comment]
I cherry-pick the HR Dataset from user-docs branch so you will see that file duplicate @hainv-ohio

**Review by hainv-ohio — APPROVED** [review]

**Pull request merged**